src/pages/Dashboard/Packages.tsx

```
@@ -1,8 +1,20 @@
+  console.log('packages',packages)
   return (
     <div>
-      packages
+       <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-4 gap-6 py-6">
+    {packages &&
+      packages.data.map((item: any,id:string) => {
+        return (
+        <div key={id}>
+          <Card card={item} link={`/packages/${item._id}`} />
+        </div>
+      )})}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)